### PR TITLE
Allow frame delays lower then 0.1sec.

### LINKED ISSUE #54: High framerates/delay lower then 0.1s not supported.
If a gif has a high framerate where the delay is < 0.1 sec, the delay defaults to 0.1 sec.

### PR DESCRIPTION
**Pull request by rokgregoric**
Fixes #54.

### CODE REVIEW

**Comment by tanukiti1987** [comment]
@rokgregoric Looks good. But can you rebase the branch by master? because CI was failing :)

**Comment by rokgregoric** [comment]
done.

File: SwiftGifCommon/UIImage+Gif.swift [hunk 1]
```diff
@@ -110,9 +110,9 @@ extension UIImage {
                 Unmanaged.passUnretained(kCGImagePropertyGIFDelayTime).toOpaque()), to: AnyObject.self)
         }
 
-        delay = delayObject as? Double ?? 0
-
-        if delay < 0.1 {
+        if let delayObject = delayObject as? Double, delayObject > 0 {
+            delay = delayObject
+        } else {
```
Comment (tanukiti1987) [inline SwiftGifCommon/UIImage+Gif.swift]: This `else` might be not needed, isn't it?
`delay` was setting `0.1` by this line. https://github.com/swiftgif/SwiftGif/pull/55/files#diff-dfaf594922b6691bbc49ec037312c27dR92

**Comment by yzhong52** [comment]
@bahlo Can we merge this?

**Comment by bahlo** [comment]
There is still an open review question but since it‘s not affecting functionality I will merge now.

**Pull request merged**